clients/include/testing_gemm_ex.hpp

```diff
@@ -225,8 +226,21 @@ inline hipblasStatus_t testing_gemm_ex_template(const Arguments& arg)
 
             if(unit_check)
             {
-                unit_check_general<Tc>(M, N, ldc, hC_gold, hC_host);
-                unit_check_general<Tc>(M, N, ldc, hC_gold, hC_device);
+                // check for mixed precision with 16 bit input and 32 bit computation
```

Comment:
Your comment implies 32 bit computation, is Tex computation or output, as half == Tex == Ta is included.

Reply:
Tex should be computation, it defaults to output (Tc) if not specified. Below from lines 43 and 44 is the prototype for testing_gemm_ex_template:

template <typename Ta, typename Tb = Ta, typename Tc = Tb, typename Tex = Tc>
inline hipblasStatus_t testing_gemm_ex_template(const Arguments& arg)

Reply:
Both hgemm (with half input/output/computation) and gemm_ex with half input/output and float computation are failing, thus the `if()` statement including both. The comment is outdated and I've updated it in the next commit.